### DISCUSSION TITLE
Is there a way to keep models on VRAM betwen workflows?

### DISCUSSION THREAD

**Discussion by leonnn1** [comfyanonymous/ComfyUI#3617] [Q&A]
I have 6 workflows for a project I did, they can't be on the same workflow because there is user input between them, so my question is:

How can I keep all the models on the VRAM when the workflows are changed, I don't want to ever unload the models, when I say models I mean, checkpoints, Ipadapters, controlnets, loras, etc...., can someone help me with that?

**Comment by ltdrdata** [comment]
Keeping the model in VRAM is completely different from keeping it in RAM. Do you really want to keep it in VRAM?

**Reply by leonnn1** [reply]
Yes I want it to not get out of the VRAM, I already calculated the amount of VRAM needed for all the workflows to match a  4090, what I need is speed, while the heavier workflow is running it reaches 22gb of VRAM usage so it never fills up the memory.
The problem is that every time I switch workflows it loads everything back which consumes a few seconds, the idea is to store the big models so generations are faster.

I tried to use the --highvram argument but it didn't worked.
One option was the inspire node pack, but the problem with it is that it only works for the SD checkpoints.
